assume/strategies/flexable.py

```diff
@@ -255,6 +255,8 @@ def get_specific_revenue(
     else:
         price_forecast = unit.price_forecast.loc[t : t + foresight]
 
+    price_forecast = price_forecast.values
+
     possible_revenue = sum(
         market_price - marginal_cost for market_price in price_forecast
     )
```

Comment:
```suggestion
    possible_revenue = (price_forecast - marginal_cost).sum()
```

This should work here too? Please test this befor committing

Reply:
Here you are mixing up bidding and state calculations. An operational window should reflect the possible states of my power plant in absolute terms. The bidding strategy in the following step takes care of the relative aspect. Such relative formulation at the operational window calculation completely broke the bidding strategies.